hiccup.gemspec

```diff
@@ -10,19 +10,23 @@ Gem::Specification.new do |s|
   s.homepage    = "http://boblail.github.com/hiccup/"
   s.summary     = %q{A library for working with things that recur}
   s.description = %q{Hiccup mixes a-la-cart recurrence features into your data structure. It doesn't dictate the data structure, just the interface.}
-  
+
   s.rubyforge_project = "hiccup"
-  
+
   s.add_dependency "activesupport", ">= 3.2.8"
+  s.add_dependency "activemodel", ">= 3.2.8"
   s.add_dependency "builder"
-  
+
   s.add_development_dependency "ri_cal"
-  s.add_development_dependency "rails", ">= 3.2.8", "< 4.1.0"
-  s.add_development_dependency "turn"
+  s.add_development_dependency "bundler", "~> 1.10"
+  s.add_development_dependency "rake", "~> 10.0"
+  s.add_development_dependency "minitest", "~> 5.0"
```

Comment:
I think so, but it's just like a gem — and this says that if minitest 6.0 comes out, hiccup isn't _necessarily_ compatible with it.